src/ClimateCO2CycleComponent.jl

```diff
@@ -1,32 +1,33 @@
 ﻿using Mimi
+using Mimi
 
 @defcomp climateco2cycle begin
     # Anthropogenic CO2 emissions in Mt of C
     mco2 = Parameter(index=[time])
 
     # Terrestrial biosphere CO2 emissions in Mt of C
-    TerrestrialCO2 = Variable(index=[time])
+    terrestrialco2 = Variable(index=[time])
 
     # Net CO2 emissions in Mt of C
     globc = Variable(index=[time])
 
     # Carbon boxes
-    cbox = Variable(index=[time,5])
+    cbox = Variable(index=[time,cpools])
```

Comment:
I'm hitting the below error when I try to go back to using `cbox = Variable(index=[time,5])` ... I'll look into this but wanted to put this here for myself.  The error is:

```
ERROR: LoadError: LoadError: MethodError: Cannot `convert` an object of type Int64 to an object of type Symbol
This may have arisen from a call to the constructor Symbol(...),
since type constructors fall back to convert methods.
```